<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:	rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QFrame" name="Frame1">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(50, 30, 30);

border-radius: 10px 100px / 120px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button11">
         <property name="styleSheet">
          <string notr="true">background-color: red;
height: 50px;
color: green;
border-radius: 10px 100px / 120px;</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button21">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: green;</string>
         </property>
         <property name="text">
          <string>(</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button31">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: green;</string>
         </property>
         <property name="text">
          <string>)</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button41">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: green;</string>
         </property>
         <property name="text">
          <string>+</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Agent1">
      <property name="styleSheet">
       <string notr="true">color: rgb(224, 27, 36);
background-color: transparent;</string>
      </property>
      <property name="text">
       <string>[ Agent A ▼ ]</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QFrame" name="Frame2">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(30, 50, 30);

border-radius: 10px 100px / 120px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="button12">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: green;</string>
         </property>
         <property name="text">
          <string>}</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button22">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: green;</string>
         </property>
         <property name="text">
          <string>P</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button32">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: green;</string>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button42">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;
color: green;</string>
         </property>
         <property name="text">
          <string>&quot;</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 41, 41);
border-radius: 50% 20% / 10% 40%;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Agent2">
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: red;
border: 3px solid #00;</string>
      </property>
      <property name="text">
       <string>[ Agent B ▼ ]</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">color: 	rgb(100, 200, 255);</string>
      </property>
      <property name="text">
       <string>Ali Ahmad Ahmadi</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: 	rgb(120, 80, 200);</string>
      </property>
      <property name="text">
       <string>Jim Carry RAsilnove</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
